packages/pds/example.dev.env

```diff
@@ -0,0 +1,6 @@
+DB_POSTGRES_URL="postgres://bsky:yksb@localhost/pds_dev"
+DEBUG_MODE=1
+LOG_ENABLED="true"
+LOG_LEVEL=debug
+LOG_DESTINATION=1
```

Comment:
yup sounds good 👍 

I've been outputting to a log file for development so my console is less noisey. but this seems like a reasonable default

Reply:
Yeah, does need to be explicitly enabled via the `.env` file (or environment variables separately), but this will be the default for folks just getting started (running the Makefile)